### ISSUE TITLE
Could the part 2 of Open Set Domain Adaptation code be solved by NLopt and libsvm?

### ISSUE THREAD

**Issue by Ariostgx** [jindongwang/transferlearning#9]
I found that the author used NLopt package to estimate W, and use libsvm to implement the part two of AIT. Could we just use those packages and get the second step done?
Also, I found in the paper that the author is using SCIP package to solve the assignment problem. Will changing the Matlab's intlinprog package to SCIP help us get better result?

**Comment by jindongwang** [comment]
Good question. I just received the reply from the author saying the code is under review now. Within days the code will be released soon.
For the assignment problem, I think any implementations are ok, since there are enough information in the paper. The tough part is part two, where so much details are missing. Let's wait for the official code. And I shall update this repo accordingly.

Best regards,
Jindong

2017年12月22日 +0800 12:07 Ariostgx <notifications@github.com>，写道：

I found that the author used NLopt package to estimate W, and use libsvm to implement the part two of AIT. Could we just use those packages and get the second step done?
Also, I found in the paper that the author is using SCIP package to solve the assignment problem. Will changing the Matlab's intlinprog package to SCIP help us get better result?
